mobile/src/bundles/users/components/profile-form/profile-form.tsx

```diff
@@ -0,0 +1,128 @@
+            ]}
+            ListHeaderComponent={
+                <>
+                    <FormField
+                        errors={errors}
+                        label="Profile name"
+                        name="profileName"
+                        required
+                        containerStyle={globalStyles.pb25}
+                    >
+                        <Input
+                            control={control}
+                            name="profileName"
+                            placeholder='ex. "Java scripter" or ".Net hard-worker"'
+                        />
+                    </FormField>
+                    <FormField
+                        errors={errors}
+                        label="Salary expectations"
+                        name="salaryExpectations"
+                        required
+                        containerStyle={globalStyles.pb25}
+                    >
+                        <Input
+                            control={control}
+                            name="salaryExpectations"
+                            placeholder="0000"
+                            inputType="number"
+                            marker="$"
+                        />
+                    </FormField>
+                    <FormField
+                        errors={errors}
+                        label="Job title"
+                        name="jobTitle"
+                        required
+                        containerStyle={globalStyles.pb25}
+                    >
+                        <Selector options={jobTitleOptions} />
+                    </FormField>
+                    <FormField
+                        errors={errors}
+                        label="Current location"
+                        name="currentLocation"
+                        required
+                        containerStyle={globalStyles.pb25}
+                    >
+                        <Selector options={currentLocationOptions} />
+                    </FormField>
+                    <FormField
+                        errors={errors}
+                        label="Briefly tell employers about your experience"
+                        name="experience"
+                        required
+                        containerStyle={globalStyles.pb25}
+                    >
+                        <Input
+                            control={control}
+                            name="experience"
+                            placeholder="Text"
+                            numberOfLines={5}
+                            multiline={true}
+                        />
+                    </FormField>
+                </>
+            }
+            ListFooterComponent={
+                <View style={globalStyles.flexDirectionRow}>
+                    <Button label="Back" disabled style={globalStyles.mr10} />
+                    <Button label="Next" onPress={handleFormSubmit} />
+                </View>
```

Comment:
These buttons appear on other screens (I have them too in my task). Perhaps we could create a separate component for them.

Reply:
I'm not sure we need a separate component. We can pass the necessary styles to the buttons